services/distribution/src/main/java/app/coronawarn/server/services/distribution/runner/TestDataGeneration.java

```diff
@@ -182,8 +181,8 @@ private DiagnosisKey generateDiagnosisKey(long submissionTimestamp) {
   }
 
   /**
-   * Returns a random rolling start number (timestamp since when a key was active, represented by a 10 minute interval
-   * counter.) between a specific submission timestamp and the beginning of the retention period.
+   * Returns a random rolling start interval number (timestamp since when a key was active, represented by a 10 minute
```

Comment:
Not introduced by this PR, I know. But should we really be using random values in tests? Shouldn't we refactor this to a parameterized test with some values, and edge cases specifically? Open issue would be fine by me.